README.md

```diff
@@ -1 +1,47 @@
-# Portfolio
\ No newline at end of file
+[](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio!
+
+> Portfolio is a Microverse project where we get to build our own responsive portfolio by creating a webpage from a figma design template. This project will demostrate our knoledge using HTML and CSS but mostly some display properties such as flexbox, we'll also demostrate knowledge using the background property.
+
+
+![screenshot](assets/demo.png)
+
+
+## Built With
+
+- HTML
+- CSS
+
+## Live Demo
+
+[Live Demo Link](https://rawcdn.githack.com/JbirdL86/Portfolio/07c3d363d6832ffa2ebd2d0ddf109757e79abf42/index.html)
+
```

Comment:
**OPTIONAL**
You should consider pushing your recent changes to the deployment server :pray: 
Although your current live demo link is working, it seems to be missing the last changes you made (i.e. background image). 
Kindly update your live version link so it's on the same level as the features in your codebase.